cmd/helper/get_workspace_config.go

```diff
@@ -75,6 +74,9 @@ func (cmd *GetWorkspaceConfigCommand) Run(ctx context.Context, args []string) er
 	if err != nil {
 		return err
 	}
+	defer func() {
+		_ = os.RemoveAll(tmpDir)
```

Comment:
nit:  `defer os.RemoveAll(tmpDir)`

Reply:
Not sure if we have the rule in this repo but linting complains about the implicit error handling usually